README.md

```diff
@@ -1 +1,25 @@
 # Notation Action
+Github Action for `notation setup`, `notation sign` and `notation verify`.
+# Usage
+1. notation-sign
+    ```sh
+    - name: sign releasd artifact with signing plugin
+      uses: notaryproject/notation-action/sign@main
+      with:
+        plugin_name: <notation_signing_plugin_name>
+        plugin_url: <plugin_download_url>
+        key_id: <key_identifier_to_sign>
+        target_artifact_reference: <target_artifact_reference_in_remote_registry>
+        plugin_config: <plugin_defined_config>
+    ```
+    For example,
```

Comment:
@priteshbandi @iamsamirzon Would you want to add the AWS Signer plugin as an example?